src/App/Header/UserMenu/UserMenu.tsx

```diff
@@ -39,22 +35,9 @@ export default function UserMenu() {
   if (user === "loading") {
     return <LoaderRing thickness={10} classes="w-6" />;
   }
-
   return (
-    <Popover className="relative">
-      <Popover.Button className="cursor-pointer contents">
-        <Icon
-          size={24}
-          type="User"
-          className="text-blue disabled:text-navy-l2"
-        />
-      </Popover.Button>
-
-      <Menu
-        user={user}
-        signOut={() => dispatch(logout())}
-        classes="mt-2 absolute z-10 w-max right-0"
-      />
-    </Popover>
+    <Link to={appRoutes.donations} className="cursor-pointer contents">
+      <Icon size={24} type="User" className="text-blue disabled:text-navy-l2" />
```

Comment:
This is the same route that exists inside `Menu` with the "My Donations" link.
Wouldn't it make for a better UI if:
- the User icon button was removed
- the menu button was turned into the hamburger icon
? 
![image](https://github.com/AngelProtocolFinance/angelprotocol-web-app/assets/19427053/fef01b89-16d3-4087-80d6-3ea4591027e0)

Reply:
Yes, I'm leaving things as close to as is for now. Doesn't hurt too much to have the spelled out extra link inside the menu + the icon outside it. I think the idea is that the icons are quick links or shortcuts to common areas that donors will use. In the future that will point to a "Profile" page and not the "My Donations" page.